tensilelite/Tensile/TensileCreateLibrary.py

```diff
@@ -644,8 +373,8 @@ def success(kernel):
       kernelHeaderFile.close()
 
   if not globalParameters["GenerateSourcesAndExit"]:
-    codeObjectFiles += buildSourceCodeObjectFiles(CxxCompiler, kernelFiles, outputPath)
-    codeObjectFiles += getAssemblyCodeObjectFiles(kernelsToBuild, kernelWriterAssembly, outputPath)
+    codeObjectFiles += SourceCommands.buildSourceCodeObjectFiles(CxxCompiler, kernelFiles, outputPath)
```

Comment:
This is a ***really good start*** to cleaning up the code used to generate binaries. In the future, I would like to consider something like:

```suggestion
    objectFiles += toolchain.compile(kernelFiles, outputPath)
    codeObjectFiles += toolchain.link(objectFiles, outputPath)
```